haystack/__init__.py

```diff
@@ -1,20 +1,21 @@
+from importlib.metadata import PackageNotFoundError, version
+
 import django
 from django.conf import settings
 from django.core.exceptions import ImproperlyConfigured
-from pkg_resources import DistributionNotFound, get_distribution, parse_version
+from packaging.version import Version
```

Comment:
https://pypi.org/project/packaging/ is not listed as a dependency in `setup.py` .

There used to be a few Version classes in python's distutils, but that is removed in Python 3.12

Maybe add a dependency on `packaging` in setup.py ?

Reply:
Hi, thanks for the suggestion. I added it.